resources/js/Pages/BenefitsReport/AssignedBenefits.vue

```diff
@@ -297,16 +298,18 @@ function isBenefitHasCompanion(benefitId) {
                     class="w-full h-full sm:text-sm appearance-none border-none text-right p-0 px-3 m-0 ring-inset hover:bg-blumilk-25 group-hover:bg-blumilk-25 focus:bg-blumilk-25 focus:ring-2 focus:ring-blumilk-300"
                     title="Wprowadź kwotę."
                     min="0"
+                    dusk="grid-employee"
                   >
                 </td>
               </template>
               <td>
-                <div class="w-full sm:text-sm focus:ring-white m-0 text-right p-3">
+                <div class="w-full sm:text-sm focus:ring-white m-0 text-right p-3" dusk="grid-sum">
```

Comment:
Take care about this GitHub warning:

![image](https://github.com/Madoq/toby/assets/56546832/8dbd14b7-d984-4917-8d51-ea7a289323a7)